src/code/fault_n64.c

```diff
@@ -0,0 +1,838 @@
+#include "global.h"
+#include "terminal.h"
+#include "alloca.h"
+
+typedef struct FaultMgr {
+    OSThread thread;
+    char unk_1B0[0x400];
+    OSMesgQueue queue;
+    OSMesg msg;
+    s32 framebuffer;
```

Comment:
```suggestion
    void *framebuffer;
```

Reply:
I can't make `Fault_DrawRec` match with framebuffer as a void*
Here's a scratch if you wanna try: https://decomp.me/scratch/mYsrs

Reply:
Any particular reason not to make it a `u16*` atleast? https://decomp.me/scratch/0OWM2

Reply:
now `u16*`